### PR TITLE
HoC 2023 - Add Dance Party AI side-by-side announcement banners

### PR DESCRIPTION
**Pull request by kelbyhawn**
Add side-by-side announcement banners to the following pages:
- Teacher homepage, logged in - [studio.code.org/home](http://studio.code.org/home)
- Student homepage logged in - [studio.code.org/home](http://studio.code.org/home)
- Student Courses page, logged in - [studio.code.org/courses](http://studio.code.org/courses)
- Learn/Courses page, logged out - [studio.code.org/courses](http://studio.code.org/courses)

**Jira ticket:** [ACQ-1121](https://codedotorg.atlassian.net/browse/ACQ-1121)

----

## Teacher homepage, logged in - [studio.code.org/home](http://studio.code.org/home)
Screenshot 2023-11-21 at 1 17 33 PM

## Student homepage, logged in - [studio.code.org/home](http://studio.code.org/home)
Screenshot 2023-11-21 at 1 15 45 PM

## Student Courses page, logged in - [studio.code.org/courses](http://studio.code.org/courses)
Screenshot 2023-11-21 at 1 14 35 PM

## Learn/Courses page, logged out - [studio.code.org/courses](http://studio.code.org/courses)
Screenshot 2023-11-21 at 1 16 12 PM

### CODE REVIEW

File: apps/src/templates/studioHomepages/SpecialAnnouncement.jsx
Comment (kelbyhawn) [inline apps/src/templates/studioHomepages/SpecialAnnouncement.jsx:1]: I think eventually we should deprecate this `SpecialAnnouncement` component and set up the `MarketingAnnouncementBanner` component to work on the Student homepage similarly to the Teacher homepage and Student logged in courses page.

File: apps/src/templates/studioHomepages/SpecialAnnouncement.jsx [hunk 1]
```diff
@@ -21,25 +21,20 @@ export default class SpecialAnnouncement extends Component {
...
     const imageUrl =
       isTeacher && isEnglish
-        ? pegasus(
-            '/shared/images/fill-540x300/announcement/announcement_hoc2020_ai.png'
-          )
-        : pegasus(
-            '/shared/images/fill-540x300/social-media/hoc2023_social.png'
-          );
+        ? pegasus('/images/dance-hoc/dance-party-activity-ai-edition.png')
+        : pegasus('/images/dance-hoc/dance-party-activity-ai-edition.png');
```
Comment (bethanyaconnor) [inline apps/src/templates/studioHomepages/SpecialAnnouncement.jsx]: Am I tired or are this the same? Is there a reason not to simplify this?

Comment (kelbyhawn) [reply apps/src/templates/studioHomepages/SpecialAnnouncement.jsx]: They are the same — I can definitely simplify it in this PR, but [this comment](https://github.com/code-dot-org/code-dot-org/pull/55099#discussion_r1401214069) gives more context why I didn't (tl:dr is that this component should probs be deprecated so I didn't want to mess anything up trying to refactor it)

Comment (bethanyaconnor) [reply apps/src/templates/studioHomepages/SpecialAnnouncement.jsx]: ah got it -- didn't connect that comment to this

Comment (bethanyaconnor) [reply apps/src/templates/studioHomepages/SpecialAnnouncement.jsx]: do we plan to revert this change after HoC? If so, I'm fine leaving it like this, but if this is the indefinite state, then I might ask for at least a comment

Comment (kelbyhawn) [reply apps/src/templates/studioHomepages/SpecialAnnouncement.jsx]: Nope, it'll stay this way after HoC. I'll clean it up and leave a comment 🙌

Comment (bethanyaconnor) [reply apps/src/templates/studioHomepages/SpecialAnnouncement.jsx]: thank you! sorry for the nitpicking -- I just know I'd have the exact same question if I worked on this file in six months 😅

Comment (kelbyhawn) [reply apps/src/templates/studioHomepages/SpecialAnnouncement.jsx]: Haha no worries at all, I appreciate the thorough review! I just updated it to no longer use logic to determine what to show for teachers or english — can you take another look? Updates are in this commit: https://github.com/code-dot-org/code-dot-org/pull/55099/commits/fc71173dcf330170f0d095db8ae56bb0a1fdd8bf

**Review by bethanyaconnor — APPROVED** [review]
Thank you! Excited to see all these changes coming together 🎉

**Pull request merged**